docs/add-ins/metadata-for-an-outlook-add-in.md

```diff
@@ -194,14 +189,43 @@ function saveCallback() {
 }
 ```
 
+### Get custom properties using EWS or REST
 
-## See also
+To get **CustomProperties** using EWS or REST, you should first determine the name of its MAPI-based extended property. You can then get that property in the same way you would get any MAPI-based extended property.
 
-- [MAPI Property Overview](https://docs.microsoft.com/office/client-developer/outlook/mapi/mapi-property-overview)   
-- [Outlook Properties Overview](https://docs.microsoft.com/office/vba/outlook/How-to/Navigation/properties-overview)  
-- [Call web services from an Outlook add-in](web-services.md)    
-- [Properties and extended properties in EWS in Exchange](https://docs.microsoft.com/exchange/client-developer/exchange-web-services/properties-and-extended-properties-in-ews-in-exchange)    
-- [Property sets and response shapes in EWS in Exchange](https://docs.microsoft.com/exchange/client-developer/exchange-web-services/property-sets-and-response-shapes-in-ews-in-exchange)
-    
+#### How custom properties are stored on an item
+
+Custom properties set by an add-in are not equivalent to normal MAPI-based properties. Essentially the add-in APIs take all of your add-in's **CustomProperties**, serialize them as a JSON payload, then save them in a single MAPI-based extended property whose name is `cecp-<app-guid>` (`<app-guid>` is your add-in's ID) and property set GUID is `{00020329-0000-0000-C000-000000000046}`. (For more information about this object, please see [MS-OXCEXT 2.2.5 Mail App Custom Properties](https://msdn.microsoft.com/library/hh968549(v=exchg.80).aspx).) You can then use EWS or REST to get this MAPI-based property.
```

Comment:
Suggest removing the word "please" from the second-to-last sentence ("please see" -> "see").